stock_orderpoint_manual_procurement_uom/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Orderpoint Manual Procurement UoM",
+    "summary": "Glue module for stock_orderpoint_uom and "
+               "stock_orderpoint_manual_procurement",
+    "version": "11.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "depends": [
+        "stock_orderpoint_uom",
+        "stock_orderpoint_manual_procurement",
+    ],
+    "data": [],
+    "license": "AGPL-3",
+    'installable': True,
+    'application': False,
+    'auto-install': True,
```

Comment:
`auto_install`

Reply:
@guewen should I remove `auto_install` ?